docs/release-notes/2.41.2.md

```diff
@@ -0,0 +1,117 @@
+
+These changes aim to improve user experience while adhering to PMM standards, particularly for managing multiple clusters.
+
+![!image](../_images/reworked_k8s_dashboard.png)
+
+
+#### Databases overview dashboard
+
+The latest release introduces a new dashboard that offers a simplified overview of the databases being monitored by PMM. This dashboard aims to provide a centralized and unified place where the crucial parameters for database performance can be easily accessed. 
+
+If you're looking for more information on this dashboard, see the [blog post]().
+
+![!image](../_images/databases_overview_dashboard.png)
+
+#### PostgreSQL Instance Summary dashboard
+
+The PostgreSQL Instance Summary dashboard is a newly designed feature that displays the critical PostgreSQL metrics. This dashboard aims to assist DBAs and developers in identifying PostgreSQL issues quickly.
+
+![!image](../_images/pg_instance_summary_dashboard.png)
+
+
+
+#### PostgreSQL Checkpoints, Buffers and WAL Usage dashboard
+
+In our continuous effort to enhance our dashboards, we have introduced this new experimental dashboard that provides more data about your PostgreSQL instances.
+
+By leveraging this data, administrators can gain insights into their PostgreSQL servers and fine-tune their performance. Understanding and managing checkpoints, buffers, and WAL usage contribute to a well-performing and reliable PostgreSQL environment.
+
+If you’re looking for in-depth insights into this dashboard, refer to our [blog post](https://www.percona.com/blog/postgresql-checkpoints-buffers-and-wal-usage-with-percona-monitoring-and-management/).
+
+![!image](../_images/pg_checkpoints_buffers_wal_dashboard.png)
+
+#### PostgreSQL Patroni dashboard
+
+Starting with PMM 2.41.2, we have included a new dashboard designed to monitor [Patroni](https://patroni.readthedocs.io/en/latest/) as an external service. For more information, see our documentation on [external services](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/external.html).
+
+This dashboard facilitates gathering more data for your PostgreSQL cluster inside PMM. 
+
+Read our [blog post](https://www.percona.com/blog/monitoring-a-postgresql-patroni-cluster/) to learn more about this dashboard and how to add Patroni monitoring.
+
+![!image](../_images/patroni_dashboard.png)
+
+!!! caution alert alert-warning "Important"
+    We would appreciate your feedback on these experimental dashboards to help us enhance it further.
+
+
+## Improvements
+
+
+[PMM-10974](https://perconadev.atlassian.net/browse/PMM-10974) - By default, PMM used `pg_stat_statements` for **stat tracking**, but now it uses `pg_stat_monitor`, which provides additional metrics and supports **explains/examples** functionality that `pg_stat_statements` does not.
```

Comment:
it actually already used to be pg_stat_monitor as default for some time already. The change we included in 2.41.2 is that when pg_stat_monitor isn't available it fallbacks to pg_stat_statements.

Reply:
Fixed it. @BupycHuk please check